manager/catalog_manager.go

```diff
@@ -328,17 +335,40 @@ func ReadTemplateVersion(catalogID string, templateID string, versionID string)
 func GetNewTemplateVersions(path string) (model.Template, bool) {
 	templateMetadata := model.Template{}
 
+	log.Debugf("Over here path : %v\n", path)
```

Comment:
Remove debugging logs statements, or make the messages something that will be meaningful if you read it in a log a month from now.